sdk/src/impl/whirlpool-client-impl.ts

```diff
@@ -101,22 +105,116 @@ export class WhirlpoolClientImpl implements WhirlpoolClient {
     );
   }
 
-  public async getPositions(positionAddresses: Address[], refresh = false): Promise<Record<string, Position | null>> {
+  public async getPositions(
+    positionAddresses: Address[],
+    refresh = false
+  ): Promise<Record<string, Position | null>> {
```

Comment:
hm. i think we are missing a pre-commit hook on this repo's git to do the yarn prettier